plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/form/form.tsx

```diff
@@ -149,6 +140,32 @@ const Form = < T extends AddressFormValues | ContactFormValues >( {
 					);
 				}
 
+				if ( field.key === 'address_1' ) {
+					return (
+						<AddressFields
+							id={ id }
+							addressType={ addressType }
+							key={ field.key }
+							address1={ addressFormFields.fields.find(
+								( customField ) =>
+									customField.key === 'address_1'
+							) }
+							address2={ addressFormFields.fields.find(
+								( customField ) =>
+									customField.key === 'address_2'
+							) }
+							addressFormFields={ addressFormFields }
+							fieldsRef={ fieldsRef }
+							onChange={ onChange }
+							values={ values }
+						/>
+					);
+				}
+
+				if ( field.key === 'address_2' ) {
+					return null;
```

Comment:
Maybe a comment here.

Reply:
That's a good suggestion. I've added comments to both sections `if ( field.key === 'address_1' ) {...}` and `if ( field.key === 'address_2' ) {...}` for clarity.